### PR TITLE
Add number support to gridXValues and gridYValues on Line

### PR DESCRIPTION
**Pull request by jasonwells**
This commit adds gridCount/integer support to `gridXValues` and `gridYValues` for line charts. This allows for grid lines and axes ticks to be calculated the same when using a number that is passed. For example:

```javascript
<ResponsiveLine
  axisRight={{
    tickValues: 6,
  }}
  gridYValues={6}
  ...
/>
```

I tried to follow all of the contributing guidelines, happy to contribute to a great library!

### CODE REVIEW

File: packages/line/src/props.js [hunk 1]
```diff
@@ -64,8 +64,14 @@ export const LinePropTypes = {
 
     enableGridX: PropTypes.bool.isRequired,
     enableGridY: PropTypes.bool.isRequired,
-    gridXValues: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
-    gridYValues: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
+    gridXValues: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
```
Comment (plouc) [inline packages/line/src/props.js]: The line chart also supports Date objects, so you should add `PropTypes.instanceOf(Date)` to both `gridXValues` and `gridYValues`.

**Comment by plouc** [comment]
@jasonwells sorry for the late review

File: packages/line/src/props.js (hunk 1 above)
Comment (jasonwells) [reply packages/line/src/props.js]: Thanks for the feedback, I wasn't sure about that. I've added it now.

**Comment by jasonwells** [comment]
@plouc Thanks for the input and no worries!

**Comment by plouc** [comment]
Thank you @jasonwells!

**Pull request merged**